tests/integration/UserLifecycleBundle/Command/BatchDeprovisionCommandTest.php

```diff
@@ -97,7 +98,9 @@ public function setUp()
         $logger = m::mock(LoggerInterface::class);
         $logger->shouldIgnoreMissing();
 
-        $this->application->add(new DeprovisionCommand($deprovisionService, $summaryService, $logger));
+        $progressReporter = new ProgressReporter();
+
+        $this->application->add(new DeprovisionCommand($deprovisionService, $summaryService, $progressReporter, $logger));
```

Comment:
Have you considered adding some assertions to the tests, ascertaining the correct progress is outputted? 

On a sidenote, I myself am guilty of not fully testing the output in the integration tests.

Reply:
The reporter has its own test. Choosing not to test the specific messages / numbers the services uses to report the progress.